src/instruments/src/SD/Pages/Apu/Apu.tsx

```diff
@@ -0,0 +1,521 @@
+                        className="GaugeText"
+                        textNudgeX={-12}
+                        textNudgeY={3}
+                        showValue
+                        bold
+                    />
+
+                    {/* AMBER BAR */}
+                    {apuEgt.isNormalOperation() && apuEgtWarning.isNormalOperation()
+                        && (
+                            <GaugeMarkerComponent
+                                x={-1}
+                                y={50}
+                                min={300}
+                                max={1100}
+                                value={apuEgtWarning.value - 33}
+                                radius={50}
+                                startAngle={240}
+                                multiplierOuter={1.2}
+                                endAngle={90}
+                                className="NoFill AmberHeavy"
+                                outer
+                            />
+                        )}
+
+                    <GaugeComponent
+                        x={0}
+                        y={50}
+                        radius={50}
+                        startAngle={240}
+                        endAngle={90}
+                        visible={redLineShown}
+                        className="Line Red NoFill"
+                    />
+                    <GaugeComponent
+                        x={0}
+                        y={50}
+                        radius={50}
+                        startAngle={240}
+                        endAngle={240 + (((Math.max(300, apuEgtWarning.value - 300)) / 800) * 210)}
+                        visible={redLineShown}
+                        className="Line White NoFill"
+                    />
+
+                    {apuEgt.isNormalOperation()
+                        && (
+                            <GaugeMarkerComponent
+                                x={0}
+                                y={50}
+                                min={300}
+                                max={1100}
+                                radius={50}
+                                startAngle={240}
+                                endAngle={90}
+                                value={apuEgt.value < 300 ? 300 : displayedEgtValue}
+                                className={`Line ${apuEgtIndicationColor === 'Pulse' ? 'LinePulse' : apuEgtIndicationColor}`}
+                                indicator
+                            />
+                        )}
+                </GaugeComponent>
+
+                <SvgGroup x={80} y={13}>
+                    <text x={0} y={0} className="White Center FontNormal">EGT</text>
+                    <text x={0} y={30} className="Cyan Center FontNormal" style={{ letterSpacing: -2 }}>&deg;C</text>
+                </SvgGroup>
+
+                <text
+                    x={10}
+                    y={80}
+                    // eslint-disable-next-line no-nested-ternary
```

Comment:
**issue**: Breaks a linting rule.
**suggestion**: Fix the code to not break the rule.
```suggestion
```